docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "2"
+
+services:
+
+    client:
+        stdin_open: true
+        tty: true
+        restart: always
+        build: .
+        depends_on:
+            - db
```

Comment:
I have tried that still got the same error.

Reply:
Actually. `db` for image `redislabs/redisgraph` is running fine because after starting it(as I can see on Portainer). If I try to run simply `querying_mechanism.py` then it works fine. There is problem only when I tried to run both combined with docker-compose.